### ISSUE TITLE
Re-structure the web modules and split some of the code into new ones

### ISSUE THREAD

**Issue by carlspring** [strongbox/strongbox#1583]
# Task Description

We need to split out the current `strongbox-web-core` into two modules in order to be able to use the real core parts of it more easily. This module has now outgrown it's original design.

Note: This task should not be done before #954 has been merged.

# Tasks

The following tasks will need to be carried out:
* [ ] Create a new aggregator called `strongbox-web`.
* [ ] Move the `strongbox-web-core` under `strongbox-web`.
* [ ] Move the `strongbox-web-forms` under `strongbox-web`.
* [ ] Create a new module called `strongbox-web-impl` under `strongbox-web`. This module should depend on the `strongbox-web-core`.
* [ ] Preserve the minimum amount of required classes under the `strongbox-web-core`.
* [ ] Move as much of the following packages to the `strongbox-web-impl`, as possible:
...(TODO: Gather a list)
* [ ] Attempt to move the sub-packages of the `org.carlspring.strongbox.controllers.layout` to their respective layout providers.
* [ ] Update the `strongbox-distribution` to include the new modules, if necessary.

# Task Relationships

This task:
* Depends on: #954 

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @fuss86
  * @sbespalov
  * @steve-todorov

**Comment by FernandaCG** [comment]
Hi, can I work on this??

**Comment by carlspring** [comment]
Hi!

No, this task is on hold. Please, feel free to join our [chat channel](https://chat.carlspring.org/) to find out what else you could help with.
